src/main/java/core/basesyntax/WorkWithBytes.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithBytes {
     public void writeBytesToFile(String fileName, byte[] data) {
-
+        try {
+            File file = new File(fileName);
+            Files.write(file.toPath(), data);
+        } catch (IOException e) {
+            System.out.println("File was not created");
```

Comment:
please throw RuntimeException instead of System.out.println
```suggestion
            throw new RuntimeException("File was not created", e);
```